src/lucky/context_extensions.cr

```diff
@@ -1,24 +1,27 @@
-require "./session/**"
-
 class HTTP::Server::Context
   DEBUG_COLOR = :green
-  setter session : Lucky::Session::AbstractStore?
```

Comment:
Having abstract store would allow using another backends, like redis for instance.

Reply:
I think we'll want that eventually, but for now we're only backing the session with cookies. When that time comes, you suggestion is excellent! 👍